src/api/projects/controllers/createProjectUser.js

```diff
@@ -0,0 +1,20 @@
+const prisma = require("../../../../prismaClient");
+
+const createProjectUser = async (req, res, next) => {
+  const { userId, projectId } = req.params;
+  try {
+    await prisma.project.update({
+      where: {
+        id: projectId,
+      },
+      data: {
+        users: { connect: { id: userId } },
+      },
+    });
+    res.sendStatus(201).json({ message: "Ok done for me projects" });
```

Comment:
You can't write anything... 😂. 
Simple that explain what was the action. Like "User added to project"

Reply:
Ok done / only sendstatus